packages/framework/ini-files/config-specs.ini

```diff
@@ -2949,6 +2951,97 @@ use RHEL7_POST
 use rhel7_sems-gnu-8.3.0-openmpi-1.10.7-openmp_release-debug_static_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables
 use PACKAGE-ENABLES|ALL
 
+[rhel8_sems-gnu-8.5.0-openmpi-4.1.6-serial_debug_shared_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables]
+#uses sems-archive modules
+use COMPILER|GNU
+use NODE-TYPE|SERIAL
+use BUILD-TYPE|DEBUG
+use LIB-TYPE|SHARED
+use KOKKOS-ARCH|NO-KOKKOS-ARCH
+
+use USE-ASAN|NO
+use USE-FPIC|NO
+use USE-MPI|YES
+use USE-PT|NO
+opt-set-cmake-var SEACASAprepro_lib_aprepro_lib_array_test_DISABLE BOOL : ON
```

Comment:
We previously turned this off for all RHEL7 builds, not sure it needs to be off for RHEL8.  May want to try leaving it on for the new batch and see if that test is still an issue.